example/ndpiReader.c

```diff
@@ -3074,6 +3093,118 @@ static void printFlowsStats() {
       }
     }
 
+    if (verbose == 4){
+		//how long the table could be
+		unsigned int len_table_max = 1000;
+	      	//number of element to delete when the table is full
+		int toDelete = 10;
+		struct hash_stats *hostsHashT = NULL;
+		struct hash_stats *host_iter = NULL;
+		struct hash_stats *tmp4 = NULL;
+		struct hash_stats *tmp5 = NULL;
+		struct hash_stats *tmp6 = NULL;
+		struct hash_stats *tmp7 = NULL;
+		int len_max = 0;    
+		      
+	      	for (i = 0; i<num_flows; i++){
+			
+		if(all_flows[i].flow->host_server_name[0] != '\0'){
+		
+			int len = strlen(all_flows[i].flow->host_server_name);
+			if(len > len_max)
+				len_max = len;
+				
+			struct hash_stats *hostFound;
+			HASH_FIND_STR(hostsHashT, all_flows[i].flow->host_server_name, hostFound);
+
+			if(hostFound == NULL){
+				struct hash_stats *newHost = (struct hash_stats*)ndpi_malloc(sizeof(hash_stats));
+			      	newHost->domain_name = all_flows[i].flow->host_server_name;
+				newHost->occurency = 1;
+				    
+				size_t size_table;
+				if ((size_table = HASH_COUNT(hostsHashT)) == len_table_max) {
```

Comment:
I've just recognized that size_table is not used. Variable and assignment can be removed, I guess.
```suggestion
				if (HASH_COUNT(hostsHashT) == len_table_max) {
```